hal/src/stm32f2xx/ota_flash_hal_stm32f2xx.cpp

```diff
@@ -616,7 +616,7 @@ const uint8_t* fetch_device_public_key(uint8_t lock)
 int HAL_Set_System_Config(hal_system_config_t config_item, const void* data, unsigned data_length)
 {
     unsigned offset = 0;
-    unsigned length = -1;
+    int length = -1;
```

Comment:
keep `unsigned length` and use `length > 0` below?

Reply:
Fixed